src/main/java/View/Web/Servlets/Public/LoginServlet.java

```diff
@@ -61,6 +61,9 @@ protected void doPost(HttpServletRequest request, HttpServletResponse response)
             HttpSession session = request.getSession();
             SessionUtil.setUser(session, ((Auth) jsonResponse.getData()).getUser());
             SessionUtil.setIsadmin(session, ((Auth) jsonResponse.getData()).getIsadmin());
+
+            // add log to database
+            (new UC_Auth(getDb())).LogLoginIntoDatabase(SessionUtil.getUser(session).getUserID(), request);
```

Comment:
super ze mame logy

Reply:
Logy ktore ti povedia 127.0.0.1 zakazdym pretože v nginx je nespravna konfiguracia ktora neberie X-forwarded-for až k nam a nedostaneme realnu ip....